test/components/calculator-process.test.js

```diff
@@ -0,0 +1,58 @@
+const calculatorProcess = require("../../src/services/calculator-process")
+
+describe("calculator's visor test", () => {
+  const operations = [
+    ["10", "+","30","=","40","C"],
```

Comment:
@Willian1661 please add just 2 more tests

```
["1.5", "+","3.5","=","5","C"],
["150", "+","250","=","400","C"],
```

Reply:
aah true, have the operations with `.` , ok

Reply:
I don't mean the operator with `.` (But should be good you create a test for this as well :) )

What I mean is you just add those 2 live in your test and see if your code is still working!

Reply:
no, the test died.
i already fixed it, but i had to create a new variable :(, i wanted just one,but it seems it can't